lib/MetaCPAN/Script/Release.pm

```diff
@@ -254,6 +263,12 @@ sub import_archive {
         $file->clear_module if ( $file->is_pod_file );
         $file->documentation;
         $file->suggest;
+
+        # check for DEPRECATED/DEPRECIATED in abstract of file
+        $file->_set_deprecated(1)
+            if $deprecated
+            or $file->abstract and $file->abstract =~ /DEPRECI?ATED/;
```

Comment:
+1 for searching for the misspelling.

Reply:
Based on the above comments, I think we need a bit more logic around flagging files as having been deprecated.  Are there any corner cases here we should note, like files containing inner packages where some could theoretically have been deprecated but others not?  Not sure how likely this is, but if it's possible, then it's worth noting.

Reply:
ok, so now the file's check for x_deprecated is in place... this is still a check for any of the file's modules being deprecated as we are marking the file and not a single module (I guess that's a fair compromise in this case).

So other files will not be affected (check using: `bin/metacpan release --latest https://cpan.metacpan.org/authors/id/E/ET/ETHER/App-Nopaste-1.011.tar.gz` - see only `Codepeek.pm` gets deprecated) 

The open question I have is - do we want to mark all the file entries as deprecated when the release is marked as deprecated?

Reply:
> The open question I have is - do we want to mark all the file entries as deprecated when the release is marked as deprecated?

I think so.